### ISSUE TITLE
Respect Named Arguments on json_decode

### ISSUE THREAD

**Issue by dcaswel** [kalessil/phpinspectionsea#1868]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) |
| Language level | PHP 8.0+                       |

Current Behavior
--
When using `json_decode()` as such:
```php
 json_decode('{"foo":"bar"}', true, flags: JSON_THROW_ON_ERROR);
```
I get a warning saying, "[EA] Please consider taking advantage of JSON_THROW_ON_ERROR flag for this call options. "

When I then click on "Harden errors handling" I get this:
```php
json_decode('{"foo":"bar"}', true, JSON_THROW_ON_ERROR, JSON_THROW_ON_ERROR);
```

Expected Behavior
--
I would expect in a project that is using PHP 8.0+ that this would not give a warning since the flag is being used.

Environment details
---
PhpStorm 2022.3
Build #PS-223.7571.212, built on December 1, 2022
Runtime version: 17.0.5+1-b653.14 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.5.1
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 16
Metal Rendering is ON
Registry:
    debugger.new.tool.window.layout=true
    run.processes.with.pty=TRUE
    ide.experimental.ui=true

Non-Bundled Plugins:
    com.your.company.unique.plugin.id (1.0)
    com.intellij.lang.puppet (223.7571.203)
    com.mallowigi (81.0.0)
    com.github.inxilpro.intellijalpine (v0.4.2)
    com.jetbrains.lang.ejs (223.7571.117)
    ru.adelf.idea.dotenv (2022.3)
    com.kalessil.phpStorm.phpInspectionsEA (4.0.7.1)
    dev.ekvedaras.laravelquery (4.0.0)
    nl.deschepers.laraveltinker (2.5.1)
    com.pestphp.pest-intellij (1.8.3)
    com.laravel_idea.plugin (6.2.2.223)
    de.espend.idea.php.annotation (8.3.0)
    fr.adrienbrault.idea.symfony2plugin (2022.1.238)

**Comment by dcaswel** [comment]
Sorry, I just saw that this is a duplicate of a couple other issues. I'll close it now.
